code/script.js

```diff
@@ -222,104 +274,133 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Chooses the secret person, randomly
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
-const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+// Lets the player confirm if they are sure before proceeding
+const guess = (secretPerson) => {
```

Comment:
It's a bit confusing naming this argument "secretPerson", since it is actually the users guess. Now it sounds like it's the actual secret person, which makes it even more confusing when we are checking the guess in checkMyGuess function. Maybe rename it to userGuess, suspect, or similar?